phoenix-core/src/it/java/org/apache/phoenix/end2end/index/MutableIndexIT.java

```diff
@@ -999,6 +1001,25 @@ public void testDeleteCount_index() throws Exception {
         }
     }
 
+    @Test
+    public void testCreateIndexSchemaVersion() throws Exception {
```

Comment:
How will the select statements behave? They need to be schema version aware at least implicitly.

Reply:
Not sure I understand this point...

Reply:
Let's say I drop the column and schema version changes from v1 to v1.1. Currently, the phoenix will use v1.1 when processing a select * .  can we add a test for that?

And is there a way I can query with old schema_version?

Reply:
Phoenix won't really "use" a schema version at all during a select. To be precise, when it grabs the PTable to be able to interpret the query results, the schema version will be in the PTable, but it's not really used by the query logic. 

The point of schema version is to allow:
1. Applications to check to see if a table, view or index is compatible with a particular version of their schema 
2. In the future, when we have change streaming from Phoenix, messages will be annotated with the schema version so consumers of the message will be able to check to see if they can interpret the message (i.e, it's compatible with a particular version of a consuming application) or not.